docker-images/kafka/scripts/kafka_tls_prepare_certificates.sh

```diff
@@ -72,19 +72,28 @@ create_keystore /tmp/kafka/cluster.keystore.p12 "$CERTS_STORE_PASSWORD" \
     "$HOSTNAME"
 echo "Preparing keystore for replication and clienttls listener is complete"
 
-CUSTOM_CERT_DIR="/opt/kafka/certificates/custom-tls-9093-certs"
-if [ -d "$CUSTOM_CERT_DIR" ]; then
-    echo "Preparing custom keystore for tls listener"
-    create_keystore_without_ca_file /tmp/kafka/custom-tls-9093.keystore.p12 "$CERTS_STORE_PASSWORD" "${CUSTOM_CERT_DIR}/tls.crt" "${CUSTOM_CERT_DIR}/tls.key" custom-key
-    echo "Preparing custom keystore for tls listener is complete"
-fi
-
-CUSTOM_CERT_DIR="/opt/kafka/certificates/custom-external-9094-certs"
-if [ -d "$CUSTOM_CERT_DIR" ]; then
-    echo "Preparing custom keystore for external listener"
-    create_keystore_without_ca_file /tmp/kafka/custom-external-9094.keystore.p12 "$CERTS_STORE_PASSWORD" "${CUSTOM_CERT_DIR}/tls.crt" "${CUSTOM_CERT_DIR}/tls.key" custom-key
-    echo "Preparing custom keystore for external listener is complete"
-fi
+regex="^\/opt\/kafka\/certificates\/(custom|oauth)-([a-z0-9]{1,11})-([0-9]+)-certs$"
+for CERT_DIR in /opt/kafka/certificates/*; do
+  if [[ $CERT_DIR =~ $regex ]]; then
+    echo "Preparing store for ${BASH_REMATCH[1]} ${BASH_REMATCH[2]} listener"
+    if [[ ${BASH_REMATCH[1]} == "custom"  ]]; then
+      echo /tmp/kafka/"${BASH_REMATCH[1]}"-"${BASH_REMATCH[2]}"-"${BASH_REMATCH[3]}".keystore.p12
```

Comment:
Should we improve this message instead of just printing the path? Or is that a debug left overs.

Reply:
Well I think it's good to have it here, as for oauth in the "Adding ... " message it mention the oauth store path, here we are printing the keystore path. We could improve but the line before says "Preparing store ... " and then the echo prints the keystore name.

Reply:
Yeah, I do not object to the log message ... but we should then improve it to not just print the path but add some explanation text.